src/cpp/exe/merge/main.cpp

```diff
@@ -8,6 +8,7 @@
 #include "antares-xpansion/benders/factories/WriterFactories.h"
 #include "antares-xpansion/benders/logger/User.h"
 #include "antares-xpansion/benders/merge_mps/MergeMPS.h"
+#include "antares-xpansion/benders/merge_mps/StandardLp.h"
```

Comment:
Unused ?

Reply:
In the current code, it is used to initialize the following static variable:
`size_t StandardLp::appendCNT = 0;`

For now I kept it, but we can find options to change it eventually